packages/fcl/src/wallet-utils/encode-provable-authn-signing.js

```diff
@@ -1,20 +1,48 @@
-import {withPrefix} from "@onflow/util-address"
+import {sansPrefix} from "@onflow/util-address"
 import {invariant} from "@onflow/util-invariant"
-import {encode} from '@onflow/rlp'
+import {encode} from "@onflow/rlp"
```

Comment:
```suggestion
import {encode as rlpEncode} from "@onflow/rlp"
```

Just a suggestion, but this makes it very clear to readers that the `encode` call below is RLP encoding.

Reply:
great suggestion